tests/test_jobqueue.py

```diff
@@ -169,6 +178,26 @@ def test_inUpdater(self):
         finally:
             u.stop()
 
+    def test_time_units(self):
```

Comment:
Please split this into multiple test that only test one thing each.
E.g. have one that tests if seconds (remember to test floats too btw) and another test which tests if datetime objects work.